### DISCUSSION TITLE
Αλλαγή ώρας για την Συμπληρωματική ΟΣΣ2 Κυριακή 10 Νοε. 6μμ

### DISCUSSION THREAD

**Discussion by PapadoAndreas** [sdy60/syllabus#169] [Show and tell]
Αν γίνεται να το κάνουμε 7:30μμ θα με βόλευε καλύτερα.

**Comment by VasilisKokkalis** [comment]
Ναι 6 ούτε εμένα εξυπηρετεί καλύτερα 7 μίση αν είναι εφικτό.

**Comment by VagNta** [comment]
Εγώ προς το παρόν είμαι ΟΚ με το 7:30 της Κυριακής.
